README.md

```
@@ -1 +1,104 @@
-# Togedy_Android_V2
+<img width="1920" height="1080" alt="togedy2" src="https://github.com/user-attachments/assets/e9f1ea4d-8cf5-4c82-ab71-11664b4410d0" />
+
+<h1 align="center">
+  <img width="72" height="72" alt="togedy_logo" src="https://github.com/user-attachments/assets/73d6678d-f4f2-4bcd-94ef-b2442e142c5e" /> Togedy</h1>
+
+<p align="center">
+  <b>Together + Study</b> — 입시생과 함께 공부하고, 입시준비를 도와주는 서비스
+</p>
+
+<p align="center">
+  <img src="https://img.shields.io/badge/Android-3DDC84?style=flat-square&logo=android&logoColor=white"/>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)